compiler/rustc_hir_typeck/src/expr.rs

```diff
@@ -2425,6 +2427,35 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         );
         err.span_label(span, format!("missing {remaining_fields_names}{truncated_fields_error}"));
 
+        if remaining_fields.items().all(|(_, (_, field))| field.value.is_some()) {
+            let msg = format!(
+                "all remaining fields have default values, {you_can} use those values with `..`",
+                you_can = if self.tcx.features().default_field_values() {
+                    "you can"
+                } else if self.tcx.sess.is_nightly_build() {
+                    "if you added `#![feature(default_field_values)]` to your crate you could"
+                } else {
+                    "if your crate were nightly-only, you could add \
+                     `#![feature(default_field_values)]` to your crate and"
+                },
```

Comment:
Discussion: we probably don't want to make the suggestion to go from a crate compilable on stable -> need nightly for this feature?